packages/regl-worldview/src/camera/camera.js

```diff
@@ -78,6 +78,14 @@ export default (regl: any) => {
         billboardRotation(context, props) {
           return selectors.billboardRotation(this.cameraState);
         },
+
+        isPerspective(context, props) {
+          return this.cameraState.perspective;
+        },
+
+        fovy(context, props) {
+          return this.cameraState.fovy;
+        },
```

Comment:
Just thinking out loud... presumably we could get these values from the `projection` value above, since that is derived from `cameraState`, is that correct? Not that we would want to do it that way.

It's curious to me that we haven't had to pass down these values before. I guess we haven't had to manipulate marker props so directly before (`scale` in this case).

Reply:
Indeed, I haven't seen those properties being used anywhere else. GLText does a lot of things differently than other commands like Cubes or Lines. 
While `fovy` can be derived from the perspective projection matrix, it does requires a bit math and I prefer not to make shaders more complex than what they are right now. `isPerspective` cannot be derived, though.